### ISSUE TITLE
User/Post Model

### ISSUE THREAD

**Issue by mctaylorpants** [xxue/ASSIGNMENT_6_BLOG_RAILS_SESSIONS#2]
This was an interesting one 😁 

So, your code is 💯% correct. The issue was this folder:

`app/evise`

This is probably left over from something else? At any rate, it contains duplicates of many of your models.

The way Rails loads your code, it will basically search through your `app/` folder and dynamically `require` all the classes you define. What was happening was it was loading the models in the `evise` folder *first*, and since it must look through the folders in alphabetical order, by the time it got to `models` it wasn't able to redefine those models with the same names.

Long story short, none of your code from `app/models/post.rb` or `app/models/user.rb` was actually being loaded 😱 

So, the solution is to delete `app/evise` and everything will work just fine!

The lesson here is to keep your codebase clean, and always be aware of what you're adding/removing with each commit you make 😄
